zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/BackupErrorResponseTest.java

```diff
@@ -126,7 +126,7 @@ void setup() {
     @Test
     void shouldReturn400() {
       // when - then
-      assertThat(backupEndpoint.status(1).getStatus()).isEqualTo(400);
```

Comment:
❓ This just changed the Java APIs right, but not the actual actuator endpoints/APIs?

Reply:
yes, the actual REST API didn't change here.